extensions/brew/src/brew.ts

```diff
@@ -240,7 +240,11 @@ export async function brewSearch(
 
     casks = casks
       ?.filter((cask) => {
-        return cask.token.toLowerCase().includes(target) || cask.desc?.toLowerCase().includes(target);
+        return (
+          cask.token.toLowerCase().includes(target) ||
+          cask.desc?.toLowerCase().includes(target) ||
+          cask.name.some((name) => name.toLowerCase().includes(target))
```

Comment:
`.some()` checks that at least one element of the array matches the condition, but since `desc` is a string, just checking it with `.includes` is fine. Good call on the order, changed it 👍🏻

Reply:
Right, I was confusing it with Swift's [Optional](https://developer.apple.com/documentation/swift/optional) which has a `some` case. Thanks for the explanation 🙏